build/Linux+BSD/portable/RecipeDebian

```diff
@@ -0,0 +1,182 @@
+      fetch-package-dependencies
+      ;;
+    "--build-only")
+      build "$@"
+      ;;
+    "--package-only")
+      package
+      ;;
+    *)
+      fetch-build-dependencies
+      fetch-package-dependencies
+      build "$@"
+      package
+  esac
+
+  echo 'RecipeDebianCross completed'
+}
+
+#################################################################################
+# Fetch compile-time dependencies only
+##################################################################################
+fetch-build-dependencies() {
+  echo "fetching dependencies for arm arch: $ARCH"
+
+  # get cross compiler toolchain
+  apt-key adv --fetch-keys http://emdebian.org/tools/debian/emdebian-toolchain-archive.key
+  echo "deb http://emdebian.org/tools/debian/ jessie main" > /etc/apt/sources.list.d/crosstools.list
+
+  # allow to download $ARCH-specific packages
+  dpkg --add-architecture $ARCH
+
+  apt-get clean && apt-get update && apt-get install -y \
+    libsndfile1-dev:$ARCH \
+    libasound2-dev:$ARCH \
+    portaudio19-dev:$ARCH \
+    zlib1g-dev:$ARCH \
+    libfreetype6-dev:$ARCH \
+    libmp3lame-dev:$ARCH \
+    libssl-dev:$ARCH \
+    libdrm-dev:$ARCH \
+    libgl1-mesa-dev:$ARCH \
+    libpulse-dev:$ARCH \
+    qtbase5-dev:$ARCH \
+    qttools5-dev:$ARCH \
+    qtscript5-dev:$ARCH \
+    libqt5xmlpatterns5-dev:$ARCH \
+    libqt5svg5-dev:$ARCH \
+    libqt5webkit5-dev:$ARCH \
+    qttools5-dev-tools:$ARCH \
+    cross-gcc-dev \
+    binutils-arm-linux-gnueabihf \
+    crossbuild-essential-armhf \
+    qemu-user-static \
+    cmake \
+    git
+
+  # forcibly install because will break aptitude dependency rules 
+  force-install qtbase5-dev-tools $ARCH
+}
+
+###################################################################################
+# Fetch run-time dependencies only...to be run after fetching compile dependencies
+###################################################################################
+fetch-package-dependencies() {
+  echo "fetching AppImage packaging + runtime dependencies for arch: $ARCH"
+
+  force-install \
+    alsa-base any \
+    alsa-tools $ARCH \
+    alsa-utils $ARCH \
+    libasound2-plugins $ARCH \
+    pulseaudio $ARCH \
+    pkg-config $ARCH \
+    liblcms2-2 $ARCH \
+    libmng1 $ARCH \
+    libicu52 $ARCH \
+    qtquick1-5-dev $ARCH \
+    libqt5quickwidgets5 $ARCH \
+    libqt5gui5 $ARCH \
+    libqt5multimedia5 $ARCH \
+    libqt5multimediawidgets5 $ARCH \
+    libqt5positioning5 $ARCH \
+    libqt5sensors5 $ARCH \
+    libqt5webkit5 $ARCH \
+    libpulse0 $ARCH \
+    libjson-c-dev $ARCH \
+    libexpat1-dev $ARCH \
+    libdbus-1-3 $ARCH \
+    libglib2.0-0 $ARCH \
+    libkeyutils-dev $ARCH \
+    libwrap0 $ARCH \
+    libacl1 $ARCH \
+    libreadline6 $ARCH \
+    libbz2-1.0 $ARCH \
+    libpcre3 $ARCH \
+    qml-module-qtquick-controls $ARCH \
+    qml-module-qtquick-dialogs $ARCH \
+    qml-module-qtquick-layouts $ARCH \
+    qml-module-qtquick2 $ARCH \
+    qml-module-qtqml-models2 $ARCH \
```

Comment:
I think we should table this issue for arm, becuase the jessie version of https://packages.debian.org/jessie/armhf/libqt5gui5/filelist doesn't have the necessary libqxcb-glx-integration.so, while the sid version does.